confdb/confdb.go

```diff
@@ -106,23 +106,23 @@ func badRequestErrorFrom(v *View, operation, request, msg string) *BadRequestErr
 	}
 }
 
-// DataBag controls access to the confdb data storage.
-type DataBag interface {
+// Databag controls access to the confdb data storage.
+type Databag interface {
 	Get(path string) (interface{}, error)
 	Set(path string, value interface{}) error
 	Unset(path string) error
 	Data() ([]byte, error)
 }
 
-// Schema takes in data from the DataBag and validates that it's valid and could
-// be committed.
-type Schema interface {
+// DatabagSchema takes in data from the Databag and validates that it's valid
+// and could be committed.
+type DatabagSchema interface {
```

Comment:
I suggest calling this StorageSchema, that's also the keyword we use for it in the assertion

Reply:
We already have a [StorageSchema](https://github.com/canonical/snapd/blob/ec7ea857712028b7e3be7a5f4448df575216dbfd/confdb/schema.go#L223), it represents a parsed schema

Reply:
ah, sorry